module/mailnews/newsread.c

```diff
@@ -12,28 +12,39 @@
 #include "client.h"
 #include "mailnews.h"
 
+#ifdef _WIN32
+#include <string.h>
+#define strcasecmp _stricmp
+#else // assuming POSIX or BSD compliant system
+#include <strings.h>
+#endif
```

Comment:
Not necessary; the client is Windows-only.  I bet you can delete this whole block (the client mostly relies on a single huge precompiled header, so individual source files rarely need to include anything extra).